src/quequations.jl

```diff
@@ -0,0 +1,54 @@
+abstract QuEquation
+
+immutable SchrodingerEquation{H<:QuBase.AbstractQuMatrix} <: QuEquation
```

Comment:
Can I convince you to rename those types again using Qu as prefix, i.e. `Qu SchrodingerEquation` (or `Qu SchrodingerEq`) and `QuLiouvillevonNeumannEquation` (or `QuLiouvillevonNeumannEq`)?